.travis.yml

```diff
@@ -28,9 +29,19 @@ matrix:
       node_js: "8"
       env: CXX=g++-4.8 TEST_SUITE=lint
     - os: linux
-      node_js: "7"
+      node_js: "8"
       env: CXX=g++-4.8 TEST_SUITE=test:node
     - os: linux
       node_js: "8"
       env: CXX=g++-4.8 TEST_SUITE=test:node
+    - os: linux
+      node_js: "10"
+      env: CXX=g++-4.8 TEST_SUITE=test:node
+    - os: linux
+    node_js: "11"
```

Comment:
Just a note in case you didn't notice yet: Travis couldn't complete - eventually some error along with the config file updates. Maybe it's this indentation here?

Reply:
This was pretty hard to debug. Thanks for noticing it :)